third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -4611,6 +4678,81 @@ resource "google_compute_node_group" "nodes" {
 `, instance, nodeTemplate, nodeGroup)
 }
 
+func testAccComputeInstance_reservationAffinity_nonSpecificReservationConfig(instanceName, reservationType string) string {
+	return fmt.Sprintf(`
+data "google_compute_image" "my_image" {
+  family  = "debian-9"
+  project = "debian-cloud"
+}
+
+resource "google_compute_instance" "foobar" {
+  name         = "%s"
+  machine_type = "n1-standard-1"
+  zone         = "us-central1-a"
+
+  boot_disk {
+    initialize_params {
+      image = data.google_compute_image.my_image.self_link
+    }
+  }
+
+  network_interface {
+    network = "default"
+  }
+
+  reservation_affinity {
+    type = "%s"
+  }
+}`, instanceName, reservationType)
+}
+
+func testAccComputeInstance_reservationAffinity_specificReservationConfig(instanceName string) string {
+	return `
```

Comment:
```suggestion
	return fmt.Sprintf(`
```

I think this is just missing a Sprintf, but it could be more! Mind double checking that this compiles+runs? Thanks!

Relevant test log lines:

```
go test  -timeout=30s $(go list ./...)
# github.com/hashicorp/terraform-provider-google-beta/v3/google-beta
google-beta/resource_compute_instance_test.go:4740:17: expected statement, found ')'
FAIL	github.com/hashicorp/terraform-provider-google-beta/v3/google-beta [setup failed]
?   	github.com/hashicorp/terraform-provider-google-beta/v3	[no test files]
?   	github.com/hashicorp/terraform-provider-google-beta/v3/scripts/affectedtests	[no test files]
?   	github.com/hashicorp/terraform-provider-google-beta/v3/scripts/sidebar	[no test files]
?   	github.com/hashicorp/terraform-provider-google-beta/v3/version	[no test files]
FAIL
```

Reply:
Ack. This was indeed just a missing `Sprintf` – had it locally as an uncommitted change. Should be good to go now – have verified compilation.